jerry-core/jerry-api.h

```diff
@@ -271,119 +270,77 @@ jerry_api_create_boolean_value (bool value);
  * with it.
  * The v_float64 member will be set and the will be JERRY_API_DATA_TYPE_FLOAT64.
  */
-extern EXTERN_C jerry_api_value_t
+jerry_api_value_t
 jerry_api_create_number_value (double value);
 
 /**
  * Creates a JERRY_API_DATA_TYPE_OBJECT type jerry_api_value_t from the
  * given jerry_api_object_t *parameter and returns with it.
  */
-extern EXTERN_C jerry_api_value_t
+jerry_api_value_t
 jerry_api_create_object_value (jerry_api_object_t *value);
 
 /**
  * Creates a JERRY_API_DATA_TYPE_STRING type jerry_api_value_t from the
  * given jerry_api_string_t *parameter and returns with it.
  */
-extern EXTERN_C jerry_api_value_t
-jerry_api_create_string_value (jerry_api_string_t *value);
+jerry_api_value_t jerry_api_create_string_value (jerry_api_string_t *value);
```

Comment:
Why did the style change here? Above this point, return type was in a separate line, from this on, it's on the same line as the rest of the function declaration. It would be good to follow one style everywhere.